lrauv_ignition_plugins/src/TethysCommPlugin.cc

```diff
@@ -260,59 +260,59 @@ void TethysCommPlugin::Configure(
 
 void TethysCommPlugin::SetupControlTopics(const std::string &_ns)
 {
-  this->thrusterTopic = ignition::transport::TopicUtils::AsValidTopic(
+  this->thrusterTopic = gz::transport::TopicUtils::AsValidTopic(
     "/model/" + _ns + "/joint/" + this->thrusterTopic);
   this->thrusterPub =
-    this->node.Advertise<ignition::msgs::Double>(this->thrusterTopic);
+    this->node.Advertise<gz::msgs::Double>(this->thrusterTopic);
   if (!this->thrusterPub)
   {
     ignerr << "Error advertising topic [" << this->thrusterTopic << "]"
       << std::endl;
   }
 
-  this->rudderTopic = ignition::transport::TopicUtils::AsValidTopic(
+  this->rudderTopic = gz::transport::TopicUtils::AsValidTopic(
     "/model/" + _ns + "/joint/" + this->rudderTopic);
   this->rudderPub =
-    this->node.Advertise<ignition::msgs::Double>(this->rudderTopic);
+    this->node.Advertise<gz::msgs::Double>(this->rudderTopic);
   if (!this->rudderPub)
   {
     ignerr << "Error advertising topic [" << this->rudderTopic << "]"
       << std::endl;
   }
 
-  this->elevatorTopic = ignition::transport::TopicUtils::AsValidTopic(
+  this->elevatorTopic = gz::transport::TopicUtils::AsValidTopic(
     "/model/" + _ns + "/joint/" + this->elevatorTopic);
   this->elevatorPub =
-    this->node.Advertise<ignition::msgs::Double>(this->elevatorTopic);
+    this->node.Advertise<gz::msgs::Double>(this->elevatorTopic);
   if (!this->elevatorPub)
   {
     ignerr << "Error advertising topic [" << this->elevatorTopic << "]"
```

Comment:
Btw, you should be able to migrate these console macros already if you want to. They aren't giving warnings right now though.